### ISSUE TITLE
git secret hide malfunctions if not run at the root of the repository

### ISSUE THREAD

**Issue by brontolinux** [sobolevn/git-secret#230]
What are the steps to reproduce this issue?
-------------------------------------------
1. have a repository with encrypted files inside a subdirectory, e.g. `my/secrets`
2. `cd` into `my/secrets`
3. run `git secret hide`

What happens?
-------------
git-secret aborts with a file not found error

What were you expecting to happen?
----------------------------------
all mapped files are encrypted again

Any logs, error output, etc?
----------------------------
```
bronto@penske:~/Downloads/project2/freeradius/3.0 (master)$ ls certs/ca.db.index*
certs/ca.db.index  certs/ca.db.index.attr  certs/ca.db.index.attr.old  certs/ca.db.index.old
bronto@penske:~/Downloads/project2/freeradius/3.0 (master)$ git secret hide -c -v

cleaning:

git-secret: abort: file not found: freeradius/3.0/certs/ca.db.index
bronto@penske:~/Downloads/project2/freeradius/3.0 (master)$ cd ../..
bronto@penske:~/Downloads/project2 (master)$ time git secret hide -v
done. all 113 files are hidden.
```

Any other comments?
-------------------
Nope

What versions of software are you using?
----------------------------------------
**Operating system:** (`uname -a`) 
Linux nelson 4.9.0-6-amd64 #1 SMP Debian 4.9.88-1+deb9u1 (2018-05-07) x86_64 GNU/Linux

**`git-secret` path:** (`which git-secret`) 
/usr/bin/git-secret

**`git-secret` version:** (`git secret --version`) 
0.2.4

**`git` version:** (`git --version`) 
git version 2.11.0

**Shell type and version:** (`$SHELL --version`) 
GNU bash, version 4.4.12(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

**`gpg` version:** (`gpg --version`) 
gpg (GnuPG) 2.1.18
libgcrypt 1.7.6-beta
Copyright (C) 2017 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Home: /home/bronto/.gnupg
Supported algorithms:
Pubkey: RSA, ELG, DSA, ECDH, ECDSA, EDDSA
Cipher: IDEA, 3DES, CAST5, BLOWFISH, AES, AES192, AES256, TWOFISH,
        CAMELLIA128, CAMELLIA192, CAMELLIA256
Hash: SHA1, RIPEMD160, SHA256, SHA384, SHA512, SHA224
Compression: Uncompressed, ZIP, ZLIB, BZIP2

**Comment by joshrabinowitz** [comment]
@brontolinux I've worked up a fix for this issue at #231

**Comment by brontolinux** [comment]
Great, thanks!
